flows/definition/results_info.go

```diff
@@ -0,0 +1,21 @@
+package definition
+
+import (
+	"github.com/nyaruka/goflow/utils"
+)
+
+// holds information about what results a flow can generate, as a map of result
+// keys to slices of result names, e.g.
+//
+//  { "age": ["Age"], "response_1": ["Response 1", "Response-1"] }
```

Comment:
From a slightly more work now but a lot simpler in the future, what do you think of this being a key -> map instead? That would let us add keys in the future (`type`?) without changing the structure of the results for all flows.

IE, something more like:
```json
{ 
  "age":  { "labels": ["Age"] },
  "response_1": { "labels": ["Response 1", "Response-1"] }
}
```